src/ui/button.ts

```diff
@@ -0,0 +1,23 @@
+import * as Phaser from 'phaser';
+import { colorPalette } from '../../assets/colorPalette';
+
+export const Button = function (scene: Phaser.Scene, x: number, y: number, text: string, maxWidth: number, onClick?: () => void) {
+  let playSoundEffects = true;
+
+  const clickEvent = () => {
+    onClick();
+    if (playSoundEffects) {
+      scene.sound.play('click');
+    }
+  }
+
+  const button = scene.add.text(x, y, text)
```

Comment:
This button is just text with a background instead of a rectangle object with a label, which should be much easier to style